package.json

```diff
@@ -64,6 +64,7 @@
     "moment": "2.10.6",
     "moment-timezone": "^0.4.0",
     "morgan": "1.2.0",
+    "ms": "0.7.1",
```

Comment:
>  Is the idea that we will start using ms everywhere that we set values in ms?

Yes, or to add a simple constants module so we can stop redefining things like `HOUR_IN_MS`